specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/examples/DetachADataDiskUsingToBeDetachedFlag.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
This example file needs to be referenced in compute.json file in the examples section.

Reply:
added